### DISCUSSION TITLE
Tell me your favorite software service (好きなソフトウェアサービスを教えてください!)

### DISCUSSION THREAD

**Discussion by sh1n00** [NAIST-SE/AttractingContributors#484] [General]
I want to know your favorite software service!
for example Notion, boost note etc. Anything is ok within related to software.
I hope you recommed nice software service to me!

My repo: [https://github.com/namekosiru/FavoriteSoftwareService](https://github.com/namekosiru/FavoriteSoftwareService)

**Comment by suika10000** [comment]
Update Lists.
visual studio code is a simple and easy-to-use editor.

If you have time, please add a note to my repository.
Here's my repo: https://github.com/suika10000/AttractingContributors-GPU

**Comment by ploychanok** [comment]
I created pull request ‘add microsoft azure’

Could you help contribute mine too?
Thank you so much
https://github.com/ploychanok/AttractingContributors-WithinThreeWords
